src/databricks/labs/ucx/install.py

```diff
@@ -807,6 +807,24 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def repair_run(self, workflow):
+        try:
+            job_id_list = [job_id for step, job_id in self._state.jobs.items() if workflow == step]
+            if job_id_list:
```

Comment:
Can you invert the logic a bit: has to be "if job id is not found, show a warning and return immediately". Don't delay that to an else statement. See contributing.md - I think i added explanation why

Reply:
inverted the logic.